devops/scripts/benchmarks/benches/benchdnn.py

```diff
@@ -0,0 +1,257 @@
+                self,
+                "graph",
+                "sdpa-plain-simplified-f16",
+                "--max-ms-per-prb=500 --dt=f32,bf16,f16 --op-kind=1:Multiply,1:Divide --case=complex_fusion/mha/sdpa-plain-simplified-f16.json",
+                eager=True,
+            ),
+            OneDnnBenchmark(
+                self,
+                "graph",
+                "sdpa-plain-wo-scale-f16-bs1",
+                "--max-ms-per-prb=500 --dt=f32,bf16,f16 --in-shapes=0:20x16x384x64+1:20x16x384x64+8:20x16x384x64+5:20x1x1x384 --case=complex_fusion/mha/sdpa-plain-wo-scale-f16-bs1.json",
+                eager=False,
+            ),
+            OneDnnBenchmark(
+                self,
+                "graph",
+                "sdpa-plain-wo-scale-f16-bs1",
+                "--max-ms-per-prb=500 --dt=f32,bf16,f16 --in-shapes=0:20x16x384x64+1:20x16x384x64+8:20x16x384x64+5:20x1x1x384 --case=complex_fusion/mha/sdpa-plain-wo-scale-f16-bs1.json",
+                eager=True,
+            ),
+        ]
+
+    def setup(self):
+        if options.sycl is None:
+            return
+
+        self.src_dir = git_clone(
+            self.directory,
+            "onednn-repo",
+            self.git_url(),
+            self.git_tag(),
+        )
+
+        self.oneapi = get_oneapi()
+        cmake_args = [
+            "cmake",
+            f"-S {self.src_dir}",
+            f"-B {self.build_dir}",
+            f"-DCMAKE_PREFIX_PATH={options.sycl}",
+            "-DCMAKE_BUILD_TYPE=Release",
+            "-DDNNL_BUILD_TESTS=ON",
+            "-DDNNL_BUILD_EXAMPLES=OFF",
+            "-DDNNL_CPU_RUNTIME=SYCL",  # Enable SYCL support
+            "-DDNNL_GPU_RUNTIME=SYCL",  # Enable SYCL GPU support
+        ]
+        run(
+            cmake_args,
+            add_sycl=True,
+        )
+
+        run(
+            f"cmake --build {self.build_dir} --target benchdnn -j {options.build_jobs}",
+            add_sycl=True,
+            ld_library=[str(self.build_dir) + "/src"] + self.oneapi.ld_libraries(),
+        )
+
+    def teardown(self):
+        pass
+
+
+class OneDnnBenchmark(Benchmark):
+    def __init__(self, suite, bench_driver, bench_name, bench_args, eager=False):
+        self.suite = suite
+        self.bench_name = f"{bench_driver}-{bench_name}"
+        self.bench_args = f"--{bench_driver} --mode=p --engine=gpu --reset {bench_args}"
+        self.exp_group = self.bench_name
+        if bench_driver == "graph":
+            if eager:
+                self.bench_args += " --execution-mode=direct"
+                self.bench_name += "-eager"
+            else:
+                self.bench_args += " --execution-mode=graph"
+                self.bench_name += "-graph"
+        self.bench_bin = suite.build_dir / "tests" / "benchdnn" / "benchdnn"
+
+    def enabled(self):
+        if options.sycl is None:
+            return False
+        if options.ur_adapter == "cuda" or options.ur_adapter == "hip":
+            return False
```

Comment:
@omarahmed1111 fyi: we are going to merge this probably with cuda/hip disabled. You might want to test and remove this condition if it turns out everything is working.